### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by rintcius**

### CODE REVIEW

**Comment by rintcius** [comment]
[ch13025]

**Comment by rintcius** [comment]
snapshot `0.2.14-a6fcb43`

**Comment by rintcius** [comment]
🛑  until all deps are non-snapshots.

File: core/src/main/scala/quasar/plugin/gcs/datasource/GCSConfig.scala [hunk 1]
```diff
@@ -16,4 +16,71 @@
...
+    authUri = EmptyUri,
+    privateKey = "",
+    clientId = "",
+    projectId = "",
+    privateKeyId = "",
+    clientEmail = "",
+    accountType = ""
+  )
+
+  val BogusAuth: ServiceAccountConfig = ServiceAccountConfig(
```
Comment (nicflores) [inline core/src/main/scala/quasar/plugin/gcs/datasource/GCSConfig.scala]: I'm going to remove this bogus auth config and only keep the one we are using in JsonSpec.scala.

**Review by nicflores — APPROVED** [review]

**Pull request merged**